impeller/renderer/backend/vulkan/command_queue_vk.cc

```diff
@@ -40,14 +39,13 @@ fml::Status CommandQueueVK::Submit(
   vk_buffers.reserve(buffers.size());
   tracked_objects.reserve(buffers.size());
   for (const std::shared_ptr<CommandBuffer>& buffer : buffers) {
-    auto encoder = CommandBufferVK::Cast(*buffer).GetEncoder();
-    if (!encoder->EndCommandBuffer()) {
+    CommandBufferVK& command_buffer = CommandBufferVK::Cast(*buffer);
```

Comment:
This should be `CommandBufferVK*`.  We are discarding the fact that this can be null.